pipelines/skylab/multiome/Multiome.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 2.3.0
```

Comment:
Would you mind updating the pipeline version in the Multiome Overview doc?

Reply:
Just bumped it! Thanks!